cpp/cmake/thirdparty/get_cucollections.cmake

```diff
@@ -14,18 +14,20 @@
 # limitations under the License.
 #=============================================================================
 
+# cuCollections doesn't have a version
+
 function(find_and_configure_cucollections)
 
     if(TARGET cuco::cuco)
```

Comment:
Now with [rapids-cmake  21.10](https://docs.rapids.ai/api/rapids-cmake/nightly/command/rapids_cpm_find.html) the `if(TARGET cuco::cuco)` check is not needed. `rapids_cpm_find` will do it internally.